posthog/models/person.py

```diff
@@ -68,6 +68,10 @@ def split_person(self, main_distinct_id: Optional[str]):
 
     # used to prevent race conditions with set and set_once
     properties_last_updated_at: models.JSONField = models.JSONField(default=dict, null=True, blank=True)
+
+    # for each prop, store if last op was $set or $set_once
```

Comment:
maybe provide the motivation instead like above, sth like
```suggestion
    # used for evaluating if we need to override the value or not (value: $set or $set_once)
```